main.go

```diff
@@ -132,6 +135,58 @@ func sendMetric(v []string, t map[string]string, message string, tags [][]string
 	fmt.Fprintf(conn, put)
 }
 
+func rejectMetric(app string, metric string, metricType string) {
+	appParts := strings.SplitN(app, "-", 2)
+	appName := appParts[0]
+	appSpace := appParts[1]
+
+	rejectMessage := struct {
+		Log        string      `json:"log"`
+		Stream     string      `json:"stream"`
+		Time       time.Time   `json:"time"`
+		Kubernetes interface{} `json:"kubernetes"`
+		Topic      string      `json:"topic"`
+	}{
+		"Unique metrics limit exceeded. Metric discarded: [" + metricType + "] " + metric,
+		"stdout",
```

Comment:
Can we add a link to somewhere (docs.akkeris.io page?) that more fully describes the issue? Some users may not understand what's going on.  Also, are you really confident `metricType` and `metric` will never contain something that may accidentally get parsed as well?

Reply:
I'll add some documentation to the Metrics docs and link it here.

As far as metricType and metric goes - they are extracted from the log message with a regex: `(measure|count|sample)#(\S*)=([0-9\.]+)(\S*)`

metricType is capture group 1 and can only be `measure`, `count`, or `sample`. 

metric is capture group 2 - and it seems like there's some possibility for a loop here: for example, if the log line was something like `measure#count#metricname=2s=2s`. I'll add some extra regex here to make sure that doesn't happen

Reply:
Replaced `#` with `_` so that we don't have any infinite loops.
Added `UNIQUE_METRIC_LIMIT_HELP` environment variable that adds a link to the docs to the rejection message